code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -251,6 +251,48 @@ datum
 				..()
 				return
 
+		harmful/hemotoxin
+			name = "hemotoxin"
+			id = "hemotoxin"
+			description = "A toxin that causes tissue damage and bleeding."
+			reagent_state = LIQUID
+			fluid_r = 195
+			fluid_g = 120
+			fluid_b = 130
+			transparency = 130
+			depletion_rate = 0.6
+
+			on_mob_life(var/mob/M, var/mult = 1)
+				if (!M) M = holder.my_atom
+				M.take_toxin_damage(0.5 * mult)
+				take_bleeding_damage(M, null, rand(5,10) * mult, DAMAGE_STAB)
+				if (prob(33))
+					M.emote(pick("groan", "pale", "scream"))
+				if (prob(25))
```

Comment:
you likely want probmult

Reply:
ok i have to be honest this is my first time coding or doing a pr what is the difference between prob and probmult

Reply:
changed to probmult